src/main/kotlin/scenario.kt

```diff
@@ -0,0 +1,14 @@
+fun scenario() {
+    simulationStep(12,8)
+}
+
+fun simulationStep(numberOfCars : Int, capacity : Int) {
+    val road = Network(capacity)
+    val cars = 1..numberOfCars
+    for (car in cars) {
+        val car = Car(true, road)
+        println("Does this car want to drive? " + car.wantsToDrive)
+        road.overCapacity()
+        println("Is this car ($car) delayed? " + road.analyzeNetwork())
```

Comment:
Given a list of cars, `analyzeNetwork` sets for each whether it is delayed or not. Once That is done, you can simply loop over that list and print for each car, which value got set.